packages/hash/api/src/model/util.ts

```diff
@@ -51,4 +59,150 @@ export const RESTRICTED_SHORTNAMES = [
   "v2",
 ];
 
-export const NIL_UUID = "00000000-0000-0000-0000-000000000000";
+export const nilUuid = "00000000-0000-0000-0000-000000000000" as const;
+
+/**
+ * @todo: create workspace types in an account that's dedciated to
+ * the HASH workspace. For now we're just chucking them in the root account.
+ */
+export const workspaceAccountId = nilUuid;
+
+export const worskspaceTypesNamespaceUri =
+  "https://example.com/@workspace/types";
+
+export const blockprotocolTypesNamespaceUri =
+  "https://blockprotocol.org/@blockprotocol/types";
```

Comment:
I think we should add a `todo` both for this and the workspace namespace to revisit (although this seems like what we concluded in the discussion about the URL for BP)

Reply:
That is true. Although the workspace namespace above could have a todo to revisit as the HASH workspace controls that, right?

Reply:
True, have added the todo in https://github.com/hashintel/hash/pull/919/commits/085bc6b4cd42caed3856be654586f77a23036cd1